project/project.properties

```diff
@@ -1,2 +1,5 @@
 ml_version=8
-nr_hosts=3
+nr_hosts=1
+priv_net_ip=192.168.33.10
```

Comment:
No worries, you can easily append to this PR by committing additional updates to your master branch..